deploy/operator.yaml

```diff
@@ -61,21 +61,13 @@ rules:
   - namespaces
   verbs:
   - list
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - get
```

Comment:
can you double check if leaseController is still working when the permission is removed? I think we might be ok with this change but want to make sure.
https://github.com/stolostron/cert-policy-controller/blob/main/main.go#L104

Reply:
That lease code from the addon-framework uses the lease API on the Hub, so access to pods shouldn't be necessary.

Reply:
@ycao56, it runs well with the updated role, but it start failing if I remove the lease/configmap permissions in the leader-election-role. Was there a specific way you wanted me to check this, or is that sufficient?

It's proving hard to test the actual deployment roles - clusters I'm using don't have the most updated helm charts already, so there would be a lot of changes to make...

Reply:
I think we are good.

Reply:
@ycao56 The pod permissions are required if a `CheckAddonPodFunc` is used in the LeaseUpdater, for example like in the status-sync: https://github.com/stolostron/governance-policy-status-sync/blob/main/main.go#L221

The cert (this one), config, and iam controllers do not use any healthCheckFuncs, so they don't currently need pod permissions. But looking at it, I'm not sure why those controllers shouldn't have a pod health check?

Reply:
That's because framework controller pod and config policy controller pod are one addon called `policy-controller`. That's why we need to use `healthCheckFuncs` to check both pods and only report it as online when both pods are running.